src/BNST2.jl

```diff
@@ -0,0 +1,30 @@
+# Biegler et al. (2000) example 2
+#
+#   Source:
+#   L. T. Biegler, J. Nocedal, C. Schmid and D. Ternet
+#   Numerical Experience with a Reduced Hessian Method for Large Scale Constrained
+#   Optimization
+#   Computational Optimization and Applications, 15(45):45-67, 2000
+#   DOI 10.1023/A:1008723031056
+#   [https://doi.org/10.1023/A:1008723031056
+#
+# A. S. Siqueira, Curitiba/BR, 01/2019.
```

Comment:
Is there a classification string we could use here?

Reply:
Well I just realized you don't seem to use classification strings in NLSProblems. Nevermind.